brave-lists/clean-urls.json

```diff
@@ -73,6 +73,37 @@
             "ref_"
         ]
     },
+    {
+        "include": [
+            "*://*.ebay.com/itm/*",
```

Comment:
Any reason why we shouldn't have the `www` like we do for the other rules?

Reply:
Some regionals; including www. also
```
https://py.ebay.com/
https://pa.ebay.com/
https://sv.ebay.com/
```

Reply:
Ok cool. We should keep the `*` then.